tests/interface/TWHDWalletTests.cpp

```diff
@@ -98,10 +98,19 @@ TEST(HDWallet, Derive) {
 TEST(HDWallet, DeriveBitcoin) {
     auto wallet = WRAP(TWHDWallet, TWHDWalletCreateWithMnemonic(words.get(), passphrase.get()));
     auto key = WRAP(TWPrivateKey, TWHDWalletGetKeyForCoin(wallet.get(), TWCoinTypeBitcoin));
-    auto publicKey = TWPrivateKeyGetPublicKeySecp256k1(key.get(), false);
+    auto publicKey = TWPrivateKeyGetPublicKeySecp256k1(key.get(), true);
```

Comment:
can we use old test case and new one?

Reply:
Right, it's always good idea to keep unaffected test cases intact. done.

Reply:
according to: https://bitcoincore.org/en/segwit_wallet_dev/ we should always use compressed public key for p2wpkh address, no need to test this IMO

Reply:
True, with non-extended key address generation does not work.  Still, often even negative or 'meaningless' tests convey some information, so I'd keep it.

Reply:
if you keep it, can you add a comment?

Reply:
already did :)